### PR TITLE
🌐 Add French translation for `docs/fr/docs/tutorial/body-multiple-params.md`

### PR DESCRIPTION
**Pull request by p3bri4n**
Bonjour 🇫🇷 🤗

Here is the PR to translate the "Body multiple params" tutorial.
See the french translation [discussion](https://github.com/tiangolo/fastapi/discussions/9185).

Thanks for the reviews 🙏

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit dd53c66fd3494b20959e7fd1ce23ce880e4aad3d at: https://18c01939.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8bad0b29a3e8d6d85f745078c78cd79d93ca58d3 at: https://6a27f490.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit c45642a39dc3aa565e12730051a2ad114e3f5afd at: https://7c0c0d31.fastapitiangolo.pages.dev

File: docs/fr/docs/tutorial/body-multiple-params.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+    },
+    "user": {
+        "username": "dave",
+        "full_name": "Dave Grohl"
+    }
+}
+```
+
+!!! note
+    Notez que malgré le fait que nous aillons déclaré le paramètre `item` de la même façon  que la fois précédente, il est désormais associé dans le body à la clé `item`.
```
Comment (GDemay) [inline docs/fr/docs/tutorial/body-multiple-params.md:97]:
```suggestion
    "Notez que, bien que nous ayons déclaré le paramètre `item` de la même manière que précédemment, il est maintenant associé à la clé `item` dans le corps de la requête."`.
```

**Review by GDemay — CHANGES_REQUESTED** [review]
Good job!

File: docs/fr/docs/tutorial/body-multiple-params.md (hunk 1 above)
Comment (pe-brian) [reply docs/fr/docs/tutorial/body-multiple-params.md]: The turn of phrase is much more natural like that, you're right, thanks !

**Comment by pe-brian** [comment]
Thank you !

**Comment by alejsdev** [comment]
Hi @GDemay, would you approve it? do you think is ready to be merged? :smile:

**Comment by GDemay** [comment]
Sounds good for me!

**Review by GDemay — APPROVED** [review]
Good, ready to merge from my perspective :)

**Comment by alejsdev** [comment]
Thank you for the help! @pe-brian @GDemay :nerd_face: :sparkles:

**Pull request merged**